src/Raven.Client/Documents/Operations/Backups/BackupResult.cs

```diff
@@ -37,6 +40,8 @@ public override DynamicJsonValue ToJson()
             json[nameof(GoogleCloudBackup)] = GoogleCloudBackup.ToJson();
             json[nameof(GlacierBackup)] = GlacierBackup.ToJson();
             json[nameof(FtpBackup)] = FtpBackup.ToJson();
+            json[nameof(FolderName)] = FolderName;
+            json[nameof(FileName)] = FileName;
```

Comment:
Shouldn't this be called `LocalBackup` to match others?

Reply:
[PeriodicBackupStatus](https://github.com/ravendb/ravendb/blob/v5.0/src/Raven.Client/Documents/Operations/Backups/PeriodicBackupStatus.cs#L43) returns FolderName

In `OneTimeBackup` we return `BackupResult` so I added `FolderName` and `FileName` to it.

Reply:
Everything else is wrapped into classes. I think we need to wrap this too.